src/Dapplo.Jira.Tests/IssueTests.cs

```diff
@@ -36,6 +37,38 @@ public async Task UpdateIssue()
 			await Client.Issue.UpdateAsync(updateIssue);
 		}
 
+		[Fact]
+		public async Task UpdateIssueLinks()
```

Comment:
This test failed with:
`IternalServerError (500). Response from server: InternalServerError`
Need to check what caused this.

I can see the content:
{"update":{"issuelinks":[{"add":{"type":{"name":"Relates","inward":"relates to","outward":"relates to"}}}]}}

Going to https://greenshot.atlassian.net/rest/api/2/issue/DIT-1?notifyUsers=True&overrideScreenSecurity=False&overrideEditableFlag=False

I'm expecting that the link type is not available in our JIRA.